src/apiCalls.js

```diff
@@ -41,11 +48,15 @@ async function createTransformer(name, description, code, language) {
         username: core.getInput("email"),
         password: core.getInput("accessToken"),
       },
+      headers: {
+        "user-agent": "transformationAction",
```

Comment:
Let's pull this header to defaultHeader value and add them to headers like this:
```javascript
headers: { ...defaultHeader }
```